app/utils/External.ts

```diff
@@ -93,7 +93,9 @@ export default class External {
                     if (response.message) {
                         reject({ message: StageUtils.resolveMessage(response.message) });
                     } else {
-                        reject({ message: undefined });
+                        // NOTE: legacy solution carried from JS
+                        // @ts-ignore property message does not exist
```

Comment:
I don't expect any changes here, but IMO this could be compressed to be more concise and elegant, and also use `@ts-expect-error` to make sure the comment is revisited in case `e.message` becomes valid:

```suggestion
                        // @ts-expect-error legacy solution carried from JS
```

Reply:
Applied